tests/functional/fixtures/ovirt.runtest/002_basic_test.py

```diff
@@ -0,0 +1 @@
+001_basic_test.py
```

Comment:
What is this change?

Reply:
It was a static file 002_basic_test.py
Changes it to a link to 001_basic_test.py